metrics/src/metrics.rs

```diff
@@ -56,11 +56,6 @@ macro_rules! datapoint {
     (@point $name:expr) => {
         $crate::influxdb::Point::new(&$name)
     };
-    ($name:expr) => {
```

Comment:
Each data point needs at least one field and since we no longer add the `host_id` field by default, this is no longer allowed